views/liff.erb

```diff
@@ -28,6 +28,17 @@
         <script type="text/javascript">
              liff.init(
                  data => {
+                     var Module = {
+                     canvas: document.getElementById('canvas')
+                         //, arguments:['palette.tic']
```

Comment:
can be deleted?

Reply:
Yes. (Also we should path a cartridge when `receiver` show this view)